net/unbound/files/root.hints

```diff
@@ -0,0 +1,39 @@
+.			518400	IN	NS	a.root-servers.net.
```

Comment:
**Update:** confirmed with Unbound team that they believe the root-servers (some-of-) will last as long as relevant device life, and Unbound does not auto update this file.